framework/encode/vulkan_state_writer.cpp

```diff
@@ -2333,6 +2333,11 @@ void VulkanStateWriter::WriteDescriptorUpdateCommand(format::HandleId
 
     const VkCopyDescriptorSet* copy = nullptr;
 
+    // scratch-space for a potential pNext-struct
+    constexpr size_t max_num_bytes_p_next_data = std::max(sizeof(VkWriteDescriptorSetAccelerationStructureKHR),
+                                                          sizeof(VkWriteDescriptorSetInlineUniformBlockEXT));
+    std::array<uint8_t, max_num_bytes_p_next_data> p_next_data{};
+
```

Comment:
@locke-lunarg this is already hinting at the [follow-up](https://github.com/LunarG/gfxreconstruct/pull/1504)

we'll do the same thing for `VkWriteDescriptorSetInlineUniformBlockEXT`